### PR TITLE
Prose

### PR DESCRIPTION
**Pull request by sophwats**
Minor prose fixes.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ In this introductory workshop, you'll learn how to use **RHODS** to recognize li
 
 And all of this without having to install anything on your own computer, thanks to **RHODS**!
 
+<!---
```
Comment (guimou) [inline docs/index.md:28]: Why do you mask the whole section?

File: docs/step3.md [hunk 2]
```diff
@@ -42,22 +42,22 @@ As you can see, you have the result of the code that was run in that cell, as we
 
 * When you save a notebook, the code as well as the results are saved! So you can always reopen it to look at the results without having to run all the program again, while still having access to the code.
 
-Therefore the name **Notebook**: it's exactly like if you were taking notes about your experiments, what you will do, with which parametets (this is the code part), and what the experiment resulted in (this is the result from a cell once it's run). Along of course with all the notes you want to take (to do that, switch the cell type from the menu from `Code`to `Markup`).
+Notebooks are so names because they are just like a physical **Notebook**: it's exactly like if you were taking notes about your experiments, (which you will do) along with the code itself, including any parameters you set. You see the output of the experiment inline (this is the result from a cell once it's run), along with all the notes you want to take (to do that, switch the cell type from the menu from `Code` to `Markup`).
```
Comment (guimou) [inline docs/step3.md]: Notebooks are so named
(and not "names")

**Review by guimou — COMMENTED** [review]
Two small things I'm not sure about.

File: docs/index.md (hunk 1 above)
Comment (sophwats) [reply docs/index.md:28]: good Q: I guess probably not, in the long term. I think by Thursday we will have something back in here - I just took it out for now incase we forget!

File: docs/step3.md (hunk 2 above)
Comment (sophwats) [reply docs/step3.md]: + good catch, updated

**Pull request merged**